ssucli/ssucli.cpp

```diff
@@ -394,6 +394,22 @@ void SsuCli::optRegister(QStringList opt)
     state = Busy;
 }
 
+void SsuCli::optUnregister(QStringList opt)
+{
+    Q_UNUSED(opt)
+    QTextStream qout(stdout);
+    QTextStream qerr(stderr);
+    QDBusPendingReply<> reply = getSsuProxy()->unregisterDevice();
+    reply.waitForFinished();
+    if (reply.isError()) {
+        qerr << fallingBackToDirectUse(reply.error()) << endl;
+        ssu.unregister();
+    }
+    qout << "Unregistered" << endl;
```

Comment:
Should the reply of unregistered only come on a successful request? It could be confusing that the request could fail even thou the message says that it was unregistered.

Reply:
Ssu::unregister() just clears out values from the configuration, so it does not fail. If it does, then there are probably bigger problems.